Description: Drop requirement for argparse
 Ubuntu and Debian both ship minimum python versions that include
 argparse; drop this from REQUIRES as its breaks the py3 unit tests
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed
Last-Update: 2015-07-09

--- unittest2-1.1.0.orig/setup.py
+++ unittest2-1.1.0/setup.py
@@ -57,7 +57,7 @@ KEYWORDS = "unittest testing tests".spli
 # Both install and setup requires - because we read VERSION from within the
 # package, and the package also exports all the APIs.
 # six for compat helpers
-REQUIRES = ['argparse', 'six>=1.4', 'traceback2'],
+REQUIRES = ['six>=1.4', 'traceback2'],
 
 params = dict(
     name=NAME,
